package.xml

```diff
@@ -10,23 +12,32 @@
   <url type="website">http://ros.org/wiki/srdfdom</url>
   <url type="bugtracker">https://github.com/ros-planning/srdfdom/issues</url>
   <url type="repository">https://github.com/ros-planning/srdfdom</url>
-  
-  <buildtool_depend>catkin</buildtool_depend>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <buildtool_depend>python_cmake_module</buildtool_depend>
+
+  <depend>console_bridge_vendor</depend>
 
   <build_depend>boost</build_depend>
-  <build_depend>cmake_modules</build_depend>
+  <build_depend>python_cmake_module</build_depend>
```

Comment:
cmake_modules wasn't related to python in ROS1. Do you really need this package? What does it do?